spec/Process/ProcessUtilsSpec.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace spec\GrumPHP\Process;
+
+use GrumPHP\Process\ProcessUtils;
+use PhpSpec\Exception\Example\SkippingException;
+use PhpSpec\ObjectBehavior;
+
+class ProcessUtilsSpec extends ObjectBehavior
+{
+    function it_is_initializable()
+    {
+        $this->shouldHaveType(ProcessUtils::class);
+    }
+
+    function it_escapes_arguments_on_windows_platform()
+    {
+        if ('\\' !== DIRECTORY_SEPARATOR) {
+            throw new SkippingException('Test requires a Windows platform.');
```

Comment:
` @requires OSFAMILY` perhaps ?

Reply:
Does that even work in phpspec? 🤔

Reply:
ah, it's phpspec? please ignore my comment then, sorry ;)

Reply:
Yeah, this one is a spec test.
The "splitted YAML linter"-stuff is phpunit, though ;)

Reply:
... which show me, that I actually messed up _big time_ in my naming for commit message over there.

Ofc it's the `symfony/yaml` component. A linter does not exist – by my knowledge. FML :(
Will fix & rebase that later tonight / this week...